dvc/stage.py

```diff
@@ -33,12 +33,12 @@ def __init__(self, fname, from_checkout=False):
         msg = "'{}' does not exist.".format(fname)
 
         sname = fname + Stage.STAGE_FILE_SUFFIX
-        prefix = "Do you mean"
+        prefix = " Do you mean"
```

Comment:
Why do we need the space in front?

Reply:
Because there is no space after the `.` in l.33

Reply:
Ah, right, sorry. Didn't notice that :)